### PR TITLE
Rubocop rake files

### PR DESCRIPTION
**Pull request by atz**

### CODE REVIEW

File: lib/tasks/pubmed.rake [hunk 1]
```diff
@@ -44,79 +49,73 @@ namespace :pubmed do
...
-        message = "*****cap_profile_id #{id} not found"
-        puts message
-        logger.error message
-      end
-      if error_count > max_errors
-        message = "Halting: Maximum number of errors #{max_errors} reached"
-        logger.error message
-        raise message
       end
+      next unless error_count > max_errors
```
Comment (darrenleeweber) [inline lib/tasks/pubmed.rake]: I don't really care about this, but I find it easier to read: `next if error_count < max_errors`

File: lib/tasks/pubmed.rake [hunk 2]
```diff
@@ -44,79 +49,73 @@ namespace :pubmed do
...
+    end_time = Time.zone.now
+    message = "Total: #{total_authors}. Authors found: #{authors_found}. Authors not found: #{authors_not_found}.  Total publications: #{total_pubs}.  Publications updated: #{pubs_updated_count}.  Errored publications: #{error_count}.  Ended at #{end_time}.  Total time: #{distance_of_time_in_words(end_time, start_time)}"
     puts message
     logger.info message
   end
 
   desc 'Retrieve and print a single publication by PubMed-ID'
   task :publication, [:pmid] => :environment do |_t, args|
-    fail "pmid argument is required." unless args[:pmid].present?
+    raise "pmid argument is required." unless args[:pmid].present?
```
Comment (darrenleeweber) [inline lib/tasks/pubmed.rake]: Prefer `if {x}.blank?` to `unless {x}.present?`

File: lib/tasks/pubmed.rake (hunk 1 above)
Comment (atz) [reply lib/tasks/pubmed.rake]: This isn't my code and wasn't flagged by rubocop.

File: lib/tasks/pubmed.rake (hunk 2 above)
Comment (atz) [reply lib/tasks/pubmed.rake]: This is a rubocop-applied revision, not something I am authoring now.  Either we can accept rubocop revisions or we can't.  There is no need to intermingle every possible desired improvement with easily approved automated and auto-prompted cleanup.

**Pull request merged**